modules/update-preparing-ack.adoc

```diff
@@ -19,9 +19,9 @@ Be aware that all responsibility falls on the administrator to ensure that all u
 
 .Procedure
 
-* Run the following command to acknowledge that you have completed the evaluation and your cluster is ready for the Kubernetes API removals in {product-title} 4.13:
+* Run the following command to acknowledge that you have completed the evaluation and your cluster is ready for the Kubernetes API removals in {product-title} 4.14:
 +
 [source,terminal]
 ----
-$ oc -n openshift-config patch cm admin-acks --patch '{"data":{"ack-4.12-kube-1.26-api-removals-in-4.13":"true"}}' --type=merge
+$ oc -n openshift-config patch cm admin-acks --patch '{"data":{"ack-4.13-kube-1.27-api-removals-in-4.14":"true"}}' --type=merge
```

Comment:
@wking Are we looking to keep with the scheme for this command for 4.14?

Reply:
@LalatenduMohanty @wking I saw in https://issues.redhat.com/browse/OTA-934 that the code for this can't be done until after an RC is available. Do we have an ETA on when this that would be? Just looking to wrap this up as soon as it's possible to. Thanks!

Reply:
@LalatenduMohanty @wking I see that OTA-934 moved to dev complete. What I have here looks to be the right value (`ack-4.13-kube-1.27-api-removals-in-4.14`). Can you PTAL and let me know if this part of the PR looks okay?

Reply:
`ack-4.13-kube-1.27-api-removals-in-4.14` looks fine. Refer `https://github.com/openshift/cluster-version-operator/pull/948/files`